index.js

```diff
@@ -17,12 +28,47 @@ class Block extends JavaScript {
     ]
   }
 
+  /**
+   * Register the plugin component.
+   *
+   * @param  {string} entry
+   * @param  {string} output
+   * @param  {object} options
+   * @return {void}
+   */
+  register(entry, output, options = {}) {
+    this.pluginOptions = (
+      options.disableRegenerator === true ? {
+        ...options,
+        requestToExternal: function(request) {
+          if (request === '@babel/runtime/regenerator') {
+            return null
+          }
+        },
+      } : options
+    )
+
+    super.register(entry, output)
+  }
+
+  /**
+   * Plugins to be merged with the master webpack config.
+   *
+   * @return {array|object}
+   */
   webpackPlugins() {
     const WordPressDependencyExtraction = require('@wordpress/dependency-extraction-webpack-plugin')
 
-    return new WordPressDependencyExtraction()
+    return new WordPressDependencyExtraction({
```

Comment:
Should probably use DependencyExtractionPlugin which you imported in the top of the file.

Reply:
😅

Reply:
Fixed in 54bd9066d2970aa34b56151797e7b47c4b0a98d6